frontend/app/utils/env.server.ts

```diff
@@ -65,14 +70,8 @@ const serverEnv = z.object({
   JAVASCRIPT_ENABLED: z.string().transform(toBoolean).default('true'),
 
   // mocks settings
-  ENABLED_MOCKS: z
-    .string() // TODO :: GjB :: can this instead be a z.enum()
-    .transform((val) => val.split(',').map((val) => val.trim()))
-    .default(''),
-  MOCK_AUTH_ALLOWED_REDIRECTS: z
-    .string()
-    .transform((val) => val.split(',').map((val) => val.trim()))
-    .default('http://localhost:3000/auth/callback/raoidc'),
+  ENABLED_MOCKS: z.string().transform(toUndefined).transform(toArray).refine(isValidMockName).default(''),
```

Comment:
`ENABLED_MOCKS` type is `string[]` so why `.default('')`?

Reply:
Same for `MOCK_AUTH_ALLOWED_REDIRECTS`.

Reply:
These are envars, which are always simple strings (by definition).

In the case of these two vars, they're comma separated values.

``` properties
ENABLED_MOCKS=foo,bar,baz
MOCK_AUTH_ALLOWED_REDIRECTS=http://localhost:3000/auth/callback,https://example.com/auth
```